docs/alert_actions/index.md

```diff
@@ -0,0 +1,138 @@
+---
+title: Alert Actions
```

Comment:
Let's link this page from here https://splunk.github.io/addonfactory-ucc-generator/#features (at least)